From 070231be40492490403790efbd7691df53f4822c Mon Sep 17 00:00:00 2001
From: Tam Nguyen <tam.nguyen.xa@renesas.com>
Date: Tue, 29 Apr 2025 17:28:23 +0700
Subject: [PATCH 36/52] configs: ironhide: Enable Ethernet 1G support for R-Car
 X5H

Enable revelant configs for Ethernet 1G support on R-Car X5H Ironhide

Connection details:
X5H/RSwitch3 <--> X5H/Ethernet PCS <--> X5H/MP-PHY <--> Ironhide/TI PHY

Signed-off-by: Tam Nguyen <tam.nguyen.xa@renesas.com>
Signed-off-by: Thanh Quan <thanh.quan.xn@renesas.com>
Signed-off-by: Hai Pham <hai.pham.ud@renesas.com>
---
 configs/renesas_rcar5.config | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/configs/renesas_rcar5.config b/configs/renesas_rcar5.config
index 780eaf7de0a..3aa387fc0ed 100644
--- a/configs/renesas_rcar5.config
+++ b/configs/renesas_rcar5.config
@@ -16,3 +16,7 @@ CONFIG_SYS_LOAD_ADDR=0x61000000
 CONFIG_SYS_RELOC_GD_ENV_ADDR=y
 CONFIG_USE_BOOTARGS=y
 CONFIG_USE_BOOTCOMMAND=y
+CONFIG_RENESAS_ETHER_SWITCH3=y
+CONFIG_PHY_R8A78000_ETHERNET_PCS=y
+CONFIG_PHY_R8A78000_MP_PHY=y
+CONFIG_PHY_TI_DP83869=y
-- 
2.34.1

